lib/attribute_normalizer/extras/version.rb

```diff
@@ -1,5 +1,5 @@
 module AttributeNormalizer
   module Extras
-    VERSION = "0.0.1"
+    VERSION = "0.1.1"
```

Comment:
This seems more like a patch-level change than a minor-version one. http://semver.org
So `0.0.2`?